qiskit_experiments/characterization/t2ramsey.py

```diff
@@ -24,7 +25,27 @@
 
 
 class T2Ramsey(BaseExperiment):
-    """T2Ramsey class"""
+
+    r"""
+    This experiment is used to estimate two properties for a single qubit:
+    T2* and Ramsey frequency.
+    The basic circuit used consists of:
+
+    #. Hadamard gate
+
+    #. Delay
+
+    #. Phase gate
+
+    #. Hadamard gate
+
+    #. Measurement
+
+    A series of such circuits is created, with increasing delays, as specified
+    by the user. The circuits are run on the device or on a simulator backend.
+    Results are analysed in the class T2RamseyAnalysis.
```

Comment:
See here https://github.com/Qiskit/qiskit-experiments/blob/main/qiskit_experiments/tomography/qst_experiment.py#L44-L65 for example formatting on how to document analysis class and options

Reply:
@chriseclectic, @nkanazawa1989 set a template in #92. Is this template identical to the format of QST, contradicts it, or adds to it?

Reply:
@chriseclectic , So for now, do we want to wait until #92 is merged, or do something temporary as in https://github.com/Qiskit/qiskit-experiments/blob/main/qiskit_experiments/tomography/qst_experiment.py#L44-L65?

Reply:
@merav-aharoni There are three separate things:
1. An issue #92 - issue, not PR - by @nkanazawa1989, with a recommendation of a standard for doc strings.
2. A PR #116, by @nkanazawa1989, where the same (or similar, not sure) standard is automated.
3. Doc strings of QST, which are probably aligned with the standard.

I don't think #116 is merged soon, @eliarbel has made some reservations. But for now we can write the doc strings according to the standard without having it automated.